.github/workflows/lua-syntax.yml

```diff
@@ -16,9 +16,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: leafo/gh-actions-lua@v8.0.0
+      - uses: leafo/gh-actions-lua@v10
         with:
-           luaVersion: "5.3.5"
+           luaVersion: "5.4"
```

Comment:
LuaJIT barely supports any 5.2 and 5.3 features. I wonder if we should test with 5.1 instead.

Reply:
Or get rid of LuaJIT and go full Lua 5.4. LuaJIT at this point is a single-maintainer project with no releases for the past 6 years. It's not like we're doing scientific computing or other CPU-heavy workloads that would benefit from it, Lua 5.4 is good enough.